openedx/core/djangoapps/user_api/models.py

```diff
@@ -31,7 +31,7 @@ class Meta(object):
         unique_together = ("user", "key")
 
     @classmethod
-    def get_value(cls, user, preference_key):
+    def get_value(cls, user, preference_key, default=None):
```

Comment:
Since you have changed the interface for this model, you should also add/update a unit test for the change.